### PR TITLE
Add Initial Support for STPA

### PR DESCRIPTION
**Pull request by danyeaw**
[System-Theoretic Process Analysis (STPA)](https://psas.scripts.mit.edu/home/get_file.php?name=STPA_handbook.pdf) is the next portion of RAAML we will support.

### PR Checklist
Please check if your PR fulfills the following requirements:

- [X] I have read, and I am following the [Contributor guide](https://github.com/gaphor/gaphor/blob/master/CONTRIBUTING.md)
- [X] I have read, and I understood the [Code of Conduct](https://github.com/gaphor/gaphor/blob/master/CODE_OF_CONDUCT.md)

### PR Type
What kind of change does this PR introduce?

- [ ] Bug fix
- [X] Feature
- [ ] Chore (refactoring, formatting, local variables, other cleanup)
- [ ] Documentation content changes

### What is the current behavior?
The RAAML profile only support FTA

Issue Number: #748 

### What is the new behavior?
RAAML now has FTA and STPA

### Does this PR introduce a breaking change?
- [ ] Yes
- [X] No

### CODE REVIEW

**Comment by danyeaw** [comment]
@amolenaar I'll submit a PR to fix distlib in MSYS2, I think this PR is ready to review when you get the chance.

File: gaphor/SysML/requirements/connectors.py [hunk 1]
```diff
@@ -46,7 +46,7 @@ class DeriveReqtConnect(DirectedRelationshipPropertyPathConnect):
 
 
 @Connector.register(Classified, RefineItem)
-class RefinefyConnect(DirectedRelationshipPropertyPathConnect):
+class RefineConnect(DirectedRelationshipPropertyPathConnect):
```
Comment (amolenaar) [inline gaphor/SysML/requirements/connectors.py:49]: If this was already in 2.4, it should also require a migration.

File: gaphor/codegen/codegen.py [hunk 2]
```diff
@@ -17,12 +17,19 @@ def main() -> None:
...
+        "--sysml_profile", help="generate a SysML profile", action="store_true"
+    )
     args = parser.parse_args()
     print(f"Generating {args.outfile} from {args.modelfile}...")
     print("  (warnings can be ignored)")
-    if args.profile:
+    if args.uml_profile:
         profile_coder.generate(args.modelfile, args.outfile, args.overrides)
+    elif args.sysml_profile:
+        profile_coder.generate(args.modelfile, args.outfile, args.overrides, True)
```
Comment (amolenaar) [inline gaphor/codegen/codegen.py:32]: `True` is more descriptive when used as keyword argument: `includes_sysml=True`.

**Review by amolenaar — CHANGES_REQUESTED** [review]
That's a lot of functionality you added!

File: gaphor/codegen/codegen.py (hunk 2 above)
Comment (danyeaw) [reply gaphor/codegen/codegen.py]: Fixed, thanks!

File: gaphor/SysML/requirements/connectors.py (hunk 1 above)
Comment (danyeaw) [reply gaphor/SysML/requirements/connectors.py:49]: I don't think the Connectors get loaded in the gaphor model files as an element right? I can load a model with a RefineItem that was saved in version 2.4.2 successfully.

Comment (amolenaar) [reply gaphor/SysML/requirements/connectors.py:49]: Yes. For connectors it fine. Sorry. I misread the decorator :see_no_evil: .

**Review by amolenaar — APPROVED** [review]

**Pull request merged**